start-local.sh

```diff
@@ -681,9 +733,98 @@ EOM
   chmod +x uninstall.sh
 }
 
+add_otel_config_in_docker_compose() {
+  # Add the OTLP configs in docker-compose.yml
+  cat >> docker-compose.yml <<-'EOM'
+configs:
+  # This is the minimal yaml configuration needed to listen on all interfaces
+  # for OTLP logs, metrics and traces, exporting to Elasticsearch.
+  edot-collector-config:
+    content: |
+      receivers:
+        otlp:
+          protocols:
+            grpc:
+              endpoint: 0.0.0.0:4317
+            http:
+              endpoint: 0.0.0.0:4318
+      
+      connectors:
+        elasticapm:
+      
+      processors:
```

Comment:
Could we add a couple of batch processor configurations? `batch` for logs and traces pipelines while `batch/metrics` for metrics pipelines, sample configuration: https://github.com/elastic/elastic-agent/blob/main/internal/pkg/otel/samples/linux/gateway.yml#L38-L44

(note that they need to be referenced in the pipeline's configuration too)